scipy/spatial/distance.py

```diff
@@ -2156,10 +2144,6 @@ def pdist(X, metric='euclidean', *, out=None, **kwargs):
         Computes the Yule distance between each pair of boolean
         vectors. (see yule function documentation)
 
-    16. ``Y = pdist(X, 'matching')``
-
-        Synonym for 'hamming'.
-
     17. ``Y = pdist(X, 'dice')``
```

Comment:
Can you update the subsequent numbers.

Reply:
Hi, sorry for the delay. I just did

Reply:
@jyuv can you please have another look? You did it in another place, but here the numbering still goes from 15 to 17